src/sentry/middleware/user.py

```diff
@@ -0,0 +1,28 @@
+from __future__ import absolute_import
+
+from datetime import timedelta
+
+from django.utils import timezone
+from sentry.api.authentication import TokenAuthentication
+
+
+class UserActiveMiddleware(object):
+    def process_view(self, request, view_func, view_args, view_kwargs):
+        try:
+            auth = TokenAuthentication()
```

Comment:
given that request.user is always set in session API calls, just remove this and then we're good to go